core/twip/balancing_twip_task.py

```diff
@@ -74,7 +80,7 @@ def __init__(
         )
 
         reward_space = gymnasium.spaces.Box(
-            low=np.array([-1.0]),
+            low=np.array([-2.0]),
```

Comment:
why -2?

Reply:
A bit below is written the penalty when you reset